components/brave_shields/core/common/shields_settings.mojom

```diff
@@ -45,3 +45,9 @@ enum HttpsUpgradeMode {
   STANDARD_MODE,
   DISABLED_MODE
 };
+
+enum AutoShredMode {
```

Comment:
per https://github.com/brave/brave-core/pull/30759/files#r2433108783 we should be very clear here that this order cannot be changed.